Automattic/wp-calypso
# Business Tools: Add categories bar and filter

Resolves https://github.com/Automattic/dotcom-forge/issues/9655

## Proposed Changes

* Adds a categories tab group to filter the cards by category

![CleanShot 2024-11-21 at 18 44 03@2x](https://github.com/user-attachments/assets/ebb33d86-664f-4632-8168-27952c847824)



## Why are these changes being made?


* To improve the usability of the Business Tools page

## Testing Instructions



* Apply this patch and navigate to `/marketing/tools/:siteId` page
* Check there is a categories tab that works as expected

## Pre-merge Checklist
